modules/catalog/src/testFixtures/java/org/apache/ignite/internal/catalog/BaseCatalogManagerTest.java

```diff
@@ -159,24 +158,19 @@ protected static CreateSortedIndexParams createSortedIndexParams(
         return createSortedIndexParams(indexName, false, indexColumns, columnsCollations);
     }
 
-    protected static CreateTableParams createTableParams(
+    protected static CatalogCommand createTableCommand(
+            CreateTableCommandBuilder builder,
```

Comment:
Personally, I don't see the point in adding a builder as a parameter and writing 40+ times `manager.createTableCommandBuilder()`  just to keep this method static, but it's up to you.

Reply:
make sense, fixed